optuna/samplers/_tpe/sampler.py

```diff
@@ -838,7 +838,11 @@ def _get_multivariate_observation_pairs(
 
         # We extract param_value from the trial.
         for param_name in param_names:
-            assert param_name in trial.params
+            assert (
+                param_name in trial.params
+            ), "Parameter {} not found in trial {}, which has parameters {}".format(
+                param_name, trial.number, list(trial.params)
+            )
```

Comment:
Looks like unrelated changes to this PR. Could you revert it?

Reply:
I reverted it to the previous expression